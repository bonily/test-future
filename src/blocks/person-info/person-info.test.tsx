import React from 'react';
import * as renderer from "react-test-renderer";
import PersonInfo from './person-info';


const person = {
  address: {
    city: `Bradford`,
    state: `TX`,
    streetAddress: `7426 Suspendisse Dr`,
    zip: `14745`,
  },
  description: `pulvinar magna convallis quis porta adipiscing mi porta vestibulum elit amet ipsum egestas tortor suspendisse mattis rutrum lacus vel libero vitae sit dolor molestie lorem sed massa lacus lacus vitae placerat et`,
  email: `MRamsey@scelerisque.com`,
  firstName: `Bonya`,
  id: 120,
  lastName: `Wittcop`,
  phone: `(731)368-3302`
};


it(`Person info should bs rendered`, () => {
  const tree = renderer
    .create(<PersonInfo
      person = {person}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

it(`Person info should bs rendered`, () => {
  const tree = renderer
    .create(<PersonInfo
      person = {undefined}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});

