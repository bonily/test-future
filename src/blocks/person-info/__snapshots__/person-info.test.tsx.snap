// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Person info should bs rendered 1`] = `
<div
  className="sc-bdfBwQ"
>
  <p
    className="sc-gsTCUz"
  >
    Выбран пользователь 
    <b>
      Bonya Wittcop
    </b>
    <br />
    Описание:
    <textarea
      readOnly={true}
      value="pulvinar magna convallis quis porta adipiscing mi porta vestibulum elit amet ipsum egestas tortor suspendisse mattis rutrum lacus vel libero vitae sit dolor molestie lorem sed massa lacus lacus vitae placerat et"
    >
      pulvinar magna convallis quis porta adipiscing mi porta vestibulum elit amet ipsum egestas tortor suspendisse mattis rutrum lacus vel libero vitae sit dolor molestie lorem sed massa lacus lacus vitae placerat et
    </textarea>
    <br />
    Адрес проживания: 
    <b>
      7426 Suspendisse Dr
    </b>
    <br />
    Город: 
    <b>
      Bradford
    </b>
    <br />
    Провинция/штат: 
    <b>
      TX
    </b>
    <br />
    Индекс: 
    <b>
      14745
    </b>
    <br />
  </p>
</div>
`;

exports[`Person info should bs rendered 2`] = `null`;
